icons/jsx/index.jsx

```diff
@@ -472,7 +472,6 @@ export { default as Check } from "./Check";
 export { default as CheckBoxIcon } from "./CheckBoxIcon";
 export { default as CheckBoxOutlineBlank } from "./CheckBoxOutlineBlank";
 export { default as CheckCircle } from "./CheckCircle";
-export { default as CheckCircleLightOutline } from "./CheckCircleLightOutline";
```

Comment:
I think we should get https://github.com/openedx/paragon/pull/2869 merged in first to avoid losing this icon

Reply:
@kalebjdavenport #2869 has been merged - would you mind rebasing this PR and ensuring `CheckCircleLightOutline` isn't removed?